site/docs/getting-started/hello-world.md

```diff
@@ -41,6 +41,17 @@ func update () {
 }
 ```
 
+```odin
+package main
+
+import "cart:w4"
```

Comment:
What is this path?! It should just be `import "w4"`

Has this example even been tested?

Reply:
I get the feeling, if this PR would be ranged between "Rushed (1)" and "Quality (10)", I'd be somewhere between 1 and 2. I'll change the status of the PR to draft and check every single code snippet once more.